### PR TITLE
OCPBUGS-14038: Update APIRemovedInNextRelease alerts

### PR DESCRIPTION
**Pull request by vrutkovs**
Alert needs to bump removed_release versions after 1.27 rebase

### CODE REVIEW

**Review by dinhxuanvu — COMMENTED** [review]
Just a quick question.

File: bindata/assets/alerts/api-usage.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
               a successful upgrade to the next EUS cluster version with Kubernetes {{ $labels.removed_release }}.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: >-
-            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1[.]2[7]"})
+            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1[.]2[89]"})
```
[the remark this answers is not in the document]
Comment (benluddy) [reply bindata/assets/alerts/api-usage.yaml:37]: It matches a single character that is either '8' or '9' (so this matches the strings "1.28" and "1.29").

[the remark this answers is not in the document]
Comment (dinhxuanvu) [reply bindata/assets/alerts/api-usage.yaml:37]: I see.

**Review by dinhxuanvu — COMMENTED** [review]
/lgtm

**Pull request merged**